### PR TITLE
Added NuGet Video "Install and Use a NuGet Package with Visual Studio"

### PR DESCRIPTION
**Pull request by chgill-MSFT**
In the preview it doesn't show the video, but it seems to appear fine when published. This is the exact type/style of embedding used in other docs with videos such as:
https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/essentials/includes/xamarin-show-essentials.md

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c23973b](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c23973bf1df1a697bd1de0da2b579872cfaaac59)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-4/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1884) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002102158244279-1884%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HOavYJa3feJEL3KJw%2B6Nl3Gwu%2FatTnJX0%2FA6%2F0rwAa0%3D&st=2020-02-10T21%3A53%3A58Z&se=2020-03-12T21%3A58%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[e9df979](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e9df979a4b8ab6d3e6445a1ca197f8ba2d1a3904)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-4/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1884) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002102201404669-1884%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3cEVTReaCpV6TXvjmnE0C3x6u5geAxAPhDpUk%2BOFrU0%3D&st=2020-02-10T21%3A57%3A15Z&se=2020-03-12T22%3A02%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/quickstart/install-and-use-a-package-in-visual-studio.md [hunk 1]
```diff
@@ -135,3 +135,9 @@ To explore more that NuGet has to offer, select the links below.
 - [Overview and workflow of package consumption](../consume-packages/overview-and-workflow.md)
 - [Finding and choosing packages](../consume-packages/finding-and-choosing-packages.md)
 - [Package references in project files](../consume-packages/package-references-in-project-files.md)
+
+## Related video
+
+> [!Video https://channel9.msdn.com/Series/NuGet-101/Install-and-Use-a-NuGet-Package-with-Visual-Studio-2-of-5]
```
Comment (zivkan) [inline docs/quickstart/install-and-use-a-package-in-visual-studio.md]: You're missing `/player` at the end of the video URL. Additionally, next time please create 1 PR with all the changes. It reduces review workload.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[dc7cb24](https://github.com/NuGet/docs.microsoft.com-nuget/commits/dc7cb2487b114cf980a9d70b5422362d0965c0fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-4/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1884) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002192125573465-1884%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ld1ozSME2C1jer8aU2jgusndRj4IYimNqR6AqqxSafA%3D&st=2020-02-19T21%3A21%3A36Z&se=2020-03-21T21%3A26%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[4b735d0](https://github.com/NuGet/docs.microsoft.com-nuget/commits/4b735d063edc586d894d67483e6cb08862e1bae9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-4/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1884) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002192204260145-1884%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9IVDCVPL10MuCuDC9KQPH4h3X83Na7aKun3C9YMZUAE%3D&st=2020-02-19T22%3A00%3A05Z&se=2020-03-21T22%3A05%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**